README_CONTAINER_IMAGE.md

```diff
@@ -47,3 +47,28 @@ Here is a detailed explanation of the options used in the command above:
 Further usage examples are available in the [examples directory](examples/) with samples of how to use the image from within OpenShift.
 
 Additional usage information for images built from `playbook2image` like this one can be found in the [playbook2image examples](https://github.com/aweiteka/playbook2image/tree/master/examples).
+
+## Running Ansible as a System Container
+
+Building the System Container: See the [BUILD.md](BUILD.md).
+
+Create `/etc/ansible` directory on the host machine and copy inventory file (hosts) into it.
```

Comment:
I agree with Rodolfo that the part about the inventory is still confusing: you don't have to put your inventory in `/etc/ansible/hosts` in the host, right? it gets bind-mounted into that path in the container, but it can come from anywhere in the host, right?